public/js/compressor.js

```diff
@@ -0,0 +1,10 @@
+export default function Compressor(audioContext) {
+  const compressor = audioContext.createDynamicsCompressor();
+  compressor.threshold.setValueAtTime(-40, audioContext.currentTime);
```

Comment:
there is some repetitive use of this value ```audioContext.currentTime```


I would store it in and the ```createDynamicsCompressor``` in two variables so you don't need to use ```audioContext``` repetitively. 

```const {createDynamicsCompressor, currentTime} = audioContext```

Reply:
Okay. Will fix this tonight.